build_runner/lib/src/generate/build_impl.dart

```diff
@@ -90,6 +91,9 @@ Future<BuildResult> build(
   final buildActions =
       await createBuildActions(targetGraph, builders, builderConfigOverrides);
 
+  validateBuilderConfig(
```

Comment:
Should we just do this as a part of `createBuildActions` so we don't need to explicitly do it (less likely to forget if we do `createBuildActions` somewhere else later on also)

Reply:
Done